star_tides/tests/fixtures/basic_user.py

```diff
@@ -13,27 +15,31 @@ def basic_user():
     created_users = []
 
     def _basic_user(
-            first_name='Test',
-            last_name='User',
-            email='test_user@example.com',
-            password='password'
+            first_name=f'Test-{gen_rand_n_str(3)}',
+            last_name=f'User-{gen_rand_n_str(3)}',
+            email=f'{gen_rand_n_str(8)}@example.com',
+            password='password',
+            kb_privilege=0
```

Comment:
Use the enum value for clarity.

Reply:
Will do. I'll be shifting the enum to the constants file.